src/Client.php

```diff
@@ -361,19 +431,6 @@ public function authenticate(string $token, string $authMethod, string $sudo = n
         $this->getHttpClientBuilder()->addPlugin(new Authentication($authMethod, $token, $sudo));
     }
 
-    /**
```

Comment:
Please revert this change.

Reply:
Reverted in d18e1d10fcd4f2768afcef81c011bcc29ce0c1e1